### ISSUE TITLE
Splitting out the tunnel directory into a separate module

### ISSUE THREAD

**Issue by arj03** [staltz/ssb-room#14]
Trying to add this as a module to ssb-browser I see that a lot of stuff is pulled in (like ssb-master). I was wondering if it would make sense to split out the actual implementation into maybe two modules? ssb-room-server and ssb-room-client?

**Comment by staltz** [comment]
> ssb-room-server and ssb-room-client?

Agree. But just to clarify, you're talking specifically about `./tunnel/server.js` and `./tunnel/client.js`, right?

**Comment by arj03** [comment]
Yes, and probably also utils.js into its own module. Still learning here :-)
